apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -15,6 +17,17 @@ class ManageStudentActionsCell extends Component {
   static propTypes = {
     id: PropTypes.number.isRequired,
     isEditing: PropTypes.bool,
+    //Provided by redux
```

Comment:
supernit: throughout most of our code, we have a space after beginning a comment (though not actually true everywhere and it doesn't actually matter that much)